apps/mobile-wallet/ios/Podfile.lock

```diff
@@ -1618,7 +1618,7 @@ PODS:
     - React-Core
   - RNFlashList (1.6.4):
     - React-Core
```

Comment:
"+" button seems cuttoff:

<img width="350" alt="image" src="https://github.com/user-attachments/assets/4d3f5f7f-db02-4a05-ad2f-a7c7160c8fd4">

Reply:
The button was removed all together. Closing.